### PR TITLE
[AIRFLOW-1621] Add tests for server side paging

### PR DESCRIPTION
**Pull request by edgarRd**
Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-1621
    - https://issues.apache.org/jira/browse/AIRFLOW-1519

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
Adding tests to check logic included in AIRFLOW-1519.

This is a follow-up PR to https://github.com/apache/incubator-airflow/pull/2531

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Tests included.

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

**Comment by edgarRd** [comment]
@bolkedebruin this is the follow up of https://github.com/apache/incubator-airflow/pull/2531

cc @aoen @saguziel

File: airflow/www/utils.py [hunk 1]
```diff
@@ -76,6 +76,20 @@ def is_accessible(self):
         )
 
 
+def get_params(**kwargs):
+    params = []
+    for k, v in kwargs.items():
+        if k == 'showPaused':
+            # True is default or None
+            if v or v is None:
```
Comment (aoen) [inline airflow/www/utils.py:84]: Should be explicit (e.g. [] vs 0 vs '      ' vs {}...)

**Comment by aoen** [comment]
Testing part LGTM although some of the other functions could use coverage as well, merging.

**Comment by bolkedebruin** [comment]
Thanks @edgarRd

**Pull request closed without merging**